src/main/java/org/fogbowcloud/manager/core/services/OrdersService.java

```diff
@@ -0,0 +1,58 @@
+package org.fogbowcloud.manager.core.services;
+
+import org.fogbowcloud.manager.core.controllers.ApplicationController;
+import org.fogbowcloud.manager.core.datastructures.SharedOrderHolders;
+import org.fogbowcloud.manager.core.exceptions.OrdersServiceException;
+import org.fogbowcloud.manager.core.models.orders.Order;
+import org.fogbowcloud.manager.core.models.orders.OrderState;
+import org.fogbowcloud.manager.core.models.token.Token;
+import org.fogbowcloud.manager.core.plugins.identity.exceptions.UnauthorizedException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import java.util.Map;
+
+public class OrdersService {
+
+    // TODO: usar log ao longo do código
```

Comment:
avoid pushing code with PT_BR comments

Reply:
Sorry for that. I am translating or resolving these comments.